### ISSUE TITLE
Aria-required property is missing for required field in Widget Builder screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#872]
**Describe the bug**
Aria-required property is missing for required field.

**To Reproduce**

1. Go to Widget Builder  (http://localhost:9992/cm/app/?view=widgetbuilder&site=test10&path=%2FSites%2Ftest10%2F&pathType=page)
2. Click on Create New Widget button. It will open a pop up.
3. "Name" is a required field. Inspect the "Name" text box element, you will find that the Aria-required=true property is missing for the input text.

**Expected behavior**
Aria-required=true property must be available for all required fields.

**Additional context**
Navigate all the options available for similar issue and fix.
